R/ptype.R

```diff
@@ -46,7 +46,7 @@
 #' @return A zero row tibble.
 #' @keywords internal
 #'
-#' @seealso [developer_functions]
+#' @seealso [developer_functions] [recipes_ptype_validate]
```

Comment:
These link to the same place, no?

Reply:
`recipes_ptype_validate()` links to `recipes_ptype()` and `recipes_ptype()` links to `recipes_ptype_validate()` 😄